anyway/views/safety_data/involved_query.py

```diff
@@ -0,0 +1,462 @@
+            return f"{vehicle_hebrew_short} - מעורב שלא נפגע"
+        elif vehicle_type is not None:
+            return f"{injured_hebrew.split(' - ')[0]} - {vehicle_hebrew_short}"
+        else:
+            return injured_hebrew
+
+
+    @staticmethod
+    def vehicle_type_bit_2_heb(bit_map: int) -> str:
+        if bit_map == 0:
+            return ""
+        res = [
+            InvolvedQuery.vehicle_type_to_str[vehicle_type][0]
+            for vehicle_type in [
+                1,
+                2,
+                3,
+                4,
+                5,
+                6,
+                7,
+                8,
+                9,
+                10,
+                11,
+                12,
+                13,
+                14,
+                15,
+                16,
+                17,
+                18,
+                19,
+                21,
+                22,
+                23,
+                24,
+                25,
+            ]
+            if bit_map & (1 << vehicle_type) != 0
+        ]
+        return ", ".join(res)
+
+    def fill_text_tables(self):
+        self.injury_severity = ["0", "הרוג", "פצוע קשה", "פצוע בינוני"]
+        self.sex = ["0", "זכר", "נקבה"]
+        self.age_group = [
+            "0",
+            "0-4",
+            "5-9",
+            "10-14",
+            "15-19",
+            "20-24",
+            "25-29",
+            "30-34",
+            "35-39",
+            "40-44",
+            "45-49",
+            "50-54",
+            "55-59",
+            "60-64",
+            "65-69",
+            "70-74",
+            "75-79",
+            "80-84",
+            "85+",
+        ]
+        self.injured_type = [
+            "0",
+            "הולך רגל",
+            "נהג - רכב בעל 4 גלגלים ויותר",
+            "נוסע - רכב בעל 4 גלגלים ויותר",
+            "נהג - אופנוע",
+            "נוסע - אופנוע (לא נהג)",
+            "נהג - אופניים",
+            "נוסע - אופניים (לא נהג)",
+            "נהג - רכב לא ידוע",
+            "נוסע - רכב לא ידוע",
+        ]
+        self.population_type = ["0", "יהודים", "ערבים", "אחרים", "זרים"]
+        self.day_in_week = ["0", "ראשון", "שני", "שלישי", "רביעי", "חמישי", "שישי", "שבת"]
+
+
+class ParamFilterExp:
+    PFE = {
+        "sy": {
+            "col": SDAccident.accident_year,
+            "op": Column.__ge__,  # default is __eq__ or in_ for a list
+            "single": True,  # default is multipe values
+        },
+        "ey": {
+            "col": SDAccident.accident_year,
+            "op": Column.__le__,
+            "single": True,  # default is multipe values
+        },
+        "sev": {
+            "col": SDInvolved.injury_severity,
+        },
+        "injt": {
+            "col": SDInvolved.injured_type,
+        },
+        "city": {
+            "col": SDAccident.accident_yishuv_symbol,
+        },
+        "st": {
+            "col": SDAccident.street1,
+        },
+        "rd": {
+            "col": SDAccident.road1,
```

Comment:
Is the filter works for accidents that their road number is either `road1` or `road2`?
Same question for street1 and street2 filters (should be "or" filter)